tox.ini

```diff
@@ -0,0 +1,29 @@
+# Note: this project is not set up for unit, functional, integration,
```

Comment:
Is everything in this file supposed to be commented out?

Reply:
Yes. See the comment at the top for the reason why.

Reply:
Guess it helps if I read. Sorry sir!

Reply:
@MJJoyce it's cool ☺️